api/tests/integration/UserTest.php

```diff
@@ -57,6 +73,7 @@ public function testGetAllPaginatedByAdminUser()
     public function testGetAllPaginatedByGuestUser()
     {
         $user = $this->createUser(['user_type' => 'guest']);
+
```

Comment:
@Redjik part of this task needs to be to remove the user type column/attribute from the user table/model respectively. This is to prove that the new roles/permission system can fulfil the previous tests.